src/server/src/districts/controllers/districts.controller.ts

```diff
@@ -59,11 +64,16 @@ export class DistrictsController {
     }
 
     const regionCode = FIPS[stateFips];
-    const regionConfig = await this.regionConfigService.findOne({
-      regionCode,
-      hidden: false,
-      archived: false
-    });
+    const regionConfig = regionConfigId
```

Comment:
Shouldn't this be the other way around?
```suggestion
    const regionConfig = regionCode
```